pages/api/wallets/user/[viewUserPubkey].ts

```diff
@@ -1,45 +1,108 @@
 // Next.js API route support: https://nextjs.org/docs/api-routes/introduction
 import type { NextApiRequest, NextApiResponse } from 'next'
-import { Membership, PrismaClient,Wallet } from '@prisma/client';
-import { MembershipModel } from '@glasseaters/hydra-sdk';
+import { Membership, PrismaClient, Wallet } from '@prisma/client';
+import { Connection, clusterApiUrl, Cluster, PublicKey } from '@solana/web3.js'
+import { FanoutClient, Fanout } from '@glasseaters/hydra-sdk';
 
-const prisma=new PrismaClient();
+const prisma = new PrismaClient();
+
+function validateMember(cluster, viewWalletPubkey) {
+  const connection = new Connection(clusterApiUrl(cluster as Cluster), 'confirmed')
+  try {
+    let publickey = new PublicKey(viewWalletPubkey);
+    console.log('testing validation');
+    let a = FanoutClient.membershipVoucher(FanoutClient.ID, publickey).then(data => { console.log(data) });
```

Comment:
I just rechecked `FanoutClient.membershipVoucher`, it calls `findProgramAddress` to get the address of the membership voucher but it does not fetch and deserialize it from the blockchain. Instead, we need to use [`FanoutMembershipVoucher.fromAccountAddress`](https://github.com/GlassEaters/hydra/blob/main/packages/sdk/src/generated/accounts/FanoutMembershipVoucher.ts#L71-L88) just like we used `Fanout.fromAccountAddress` in wallet validation. The address it takes is the membership voucher address obtained by `FanoutClient.membershipVoucher`.

Reply:
ok will change the Fanout call function